elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/SQLQueryEngine.java

```diff
@@ -60,13 +62,15 @@
 @Slf4j
 public class SQLQueryEngine extends QueryEngine {
     private final EntityManagerFactory entityManagerFactory;
-
+    private final TransactionCancel transactionCancel;
     private final SQLReferenceTable referenceTable;
+    private EntityManager entityManager;
 
-    public SQLQueryEngine(MetaDataStore metaDataStore, EntityManagerFactory entityManagerFactory, Cache cache) {
-        super(metaDataStore, cache);
-        this.entityManagerFactory = entityManagerFactory;
-        this.referenceTable = new SQLReferenceTable(metaDataStore);
+    public SQLQueryEngine(MetaDataStore mDStore, EntityManagerFactory emFactory, Cache cx, TransactionCancel txCancel) {
```

Comment:
cache is much more readable than cx.  Can you revert back to the more readable variable names?

Reply:
Same for mDStore, qEngine, emFactory, etc.